src/sections/Community/Handbook/writing-program/content-form.style.js

```diff
@@ -29,7 +29,12 @@ const ContentFormWrapper = styled.section`
         font-size: .9rem;
         margin: 0 1rem 1rem;
         padding: 1rem;
-        
+        overflow: auto;
+        height: 600px;
```

Comment:
Is specification of `height` necessary?

Reply:
Yes. From my understanding, for overflow to happen, the height has to be set so that anything which cannot be fit in the specified limit will be the content that gets the scroll option.